conversation/csharp/sdk/conversation/conversation.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
Do we need net 9?  I'm thinking we could stay on .NEt 8 because it's LTS, it's on more dev boxes, and that's whats in our test env unless we take action to change the GH actions.

Reply:
Could you also please sync to `release-1.15` and attempt to get C# checks to pass?

Reply:
We can stay on .NET 8 here if you'd like. We've got support for .NET 6, 8 and 9 with 1.15 for all clients except Dapr.Workflow (which supports .NET 7, 8 and 9). Version 1.16 will drop .NET 6 and .NET 7.